### DISCUSSION TITLE
Feature Request: Export-Feed - Format date

### DISCUSSION THREAD

**Discussion by ghost** [atrocore/atrocore#685] [Ideas]
It would be amazing to be able to define a format for a date(-time) field during export; So it would be possible to match the "needed" format for other third party system. 

Example:
Price valid from will be exported as saved in the database like 2023-01-01 but system X needs 20230101 and system Y needs 2023/01/01 etc...

**Comment by atrolex** [comment]
It is already possible with the Connector module. You can write twig scripts to do any transformations to your data, not only to datatime.
